ui/src/app/views/account-creation-success/component.tsx

```diff
@@ -0,0 +1,134 @@
+import {Component, DoCheck, Input, OnInit, ViewChild} from '@angular/core';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {SignInService} from 'app/services/sign-in.service';
+
+import {
+  BolderHeader,
+  Header,
+  SmallHeader
+} from 'app/common/common';
+import {AccountCreationModalsComponent} from 'app/views/account-creation-modals/component';
+import {AccountCreationComponent} from 'app/views/account-creation/component';
+import {LoginComponent} from 'app/views/login/component';
+
+const buttonLinkStyling = {
+  border: 'none',
+  cursor: 'pointer',
+  outlineColor: 'transparent',
+  color: '#2691D0',
+  backgroundColor: 'transparent',
+};
+
+export class AccountCreationSuccessReact extends React.Component<any, any> {
```

Comment:
should define the type of the state and props and pass it in here instead of using any

Reply:
I thought I'd pushed that up... might just be local. Will push after other changes have been addressed.